src/cache/cache_las.c

```diff
@@ -1318,10 +1302,10 @@ __wt_las_sweep(WT_SESSION_IMPL *session)
         }
 
         /*
-         * There are several conditions that need to be met before we choose to remove a lookaside
-         * entry:
-         *  1. If there exists a last checkpoint timestamp, then the lookaside record timestamp must
-         *     be less than last checkpoint timestamp.
+         * There are several conditions that need to be met before we choose to remove a
```

Comment:
s_all run changed the format of these comments. I don't know how to avoid that. Any ideas?